### PR TITLE
Run tests in Firefox on Travis

### PR DESCRIPTION
**Pull request by forbesjo**
## Description
Use Firefox along with Chrome for the Travis builds.

## Requirements Checklist
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "lint": "vjsstandard",
     "start": "npm-run-all -p watch start:*",
     "start:serve": "babel-node scripts/server.js",
-    "pretest": "npm-run-all lint build",
+    "pretest": "npm-run-all build",
     "test": "karma start test/karma/detected.js",
```
Comment (BrandonOCasey) [inline package.json]: I think there was talk of doing posttest linting rather than pretest linting

Comment (forbesjo) [inline package.json]: I'm just disabling temporarily in order to get karma to run. When the linting issues are resolved I'll rebase and remove the commit.

Comment (BrandonOCasey) [inline package.json]: ahh ok

**Comment by BrandonOCasey** [comment]
linting should be resolved by #788

**Pull request closed without merging**